ports/stm32/boards/ARDUINO_GIGA/pins.csv

```diff
@@ -1,3 +1,115 @@
+D0,PB7
```

Comment:
Please shift these down to after `PK7`, ie just before the UART definitions begin.

Also, please describe them with a comment so it's clear what they are for, eg:
```
PK7,PK7

# Arduino digital/analog pins
D1,PA9
...
```